### DISCUSSION TITLE
Pacific alerts - Mexico coast

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#553] [Q&A]
There is currently a hurricane warning issued for PMZ027. This doesn't show up in the API.

   https://api.weather.gov/alerts/active/zone/PMZ027

returns no current alerts.

Is this another case of something that is not injested by the API yet?

**Comment by akrherz** [comment]
Do you have a link to the text product that contains this warning?

**Comment by JJKraw** [comment]
https://alerts.weather.gov/cap/wwacapget.php?x=MZPI1263F564DD20.HurricaneWarning.1263F566B62CMZPI.NHCOFFPZ7.64487d73bd5a451c3d1833deaa7e3af3

**Comment by JJKraw** [comment]
Bumping this. Just want to know what the situation is with these particular marine zones. Thanks!

**Reply by jah1007** [reply]
The upstream application which produces the CAP 1.2 formatted products for API does not have the same requirements as the CAP 1.1 (alerts.weather.gov) site. Since the Hurricane Warning in question was produced from an Offshore Forecast (generated by the National Hurricane Center) and not a WFO, API would not carry this warning.
